Cod3rsGrowth.Teste/Testes/BaralhoTest.cs

```diff
@@ -0,0 +1,261 @@
+                                TipoDeCarta = TipoDeCartaEnum.TerrenoBasico,
+                                RaridadeCarta = RaridadeEnum.Common,
+                                PrecoCarta = Convert.ToDecimal(0.5),
+                                CorCarta = new List<CoresEnum>() { CoresEnum.Incolor }
+                            },
+                            QuantidadeCopiasDaCartaNoBaralho = 49
+                        },
+                        new CopiaDeCartasNoBaralho ()
+                        {
+                            Carta = new Carta()
+                            {
+                                IdCarta = 5,
+                                NomeCarta = "Montanha",
+                                CustoDeManaConvertidoCarta = 0,
+                                TipoDeCarta = TipoDeCartaEnum.TerrenoBasico,
+                                RaridadeCarta = RaridadeEnum.Common,
+                                PrecoCarta = Convert.ToDecimal(0.5),
+                                CorCarta = new List<CoresEnum>() { CoresEnum.Incolor }
+                            },
+                            QuantidadeCopiasDaCartaNoBaralho = 50
+                        },
+                        new CopiaDeCartasNoBaralho ()
+                        {
+                            Carta = new Carta()
+                            {
+                                IdCarta = 8,
+                                NomeCarta = "Niv-Mizzet, Parum",
+                                CustoDeManaConvertidoCarta = 6,
+                                TipoDeCarta = TipoDeCartaEnum.Criatura,
+                                RaridadeCarta = RaridadeEnum.Rare,
+                                PrecoCarta = Convert.ToDecimal(5),
+                                CorCarta = new List<CoresEnum>() { CoresEnum.Azul, CoresEnum.Vermelho }
+                            },
+                            QuantidadeCopiasDaCartaNoBaralho = 1
+                        }
+                    },
+                    QuantidadeDeCartasNoBaralho = 100,
+                    PrecoDoBaralho = 54.5m,
+                    CustoDeManaConvertidoDoBaralho = 6,
+                    CorBaralho = new List<CoresEnum>() {CoresEnum.Azul, CoresEnum.Vermelho}
+                }
+            };
+
+            if(ObterServico.ObterTodos().Count() < 1) listaBaralhosMock.ForEach(baralho => ObterServico.Inserir(baralho));
+        }
+
+        [Fact]
+        public void verifica_se_a_lista_nao_esta_vazia()
+        {
+            var baralhos = ObterServico.ObterTodos();
+
+            Assert.NotEmpty(baralhos);
+        }
+
+        [Fact]
+        public void ao_ObterTodos_deve_retornar_uma_lista_com_dois_baralhos()
+        {
+
+            var quantidadeDeBaralhos = ObterServico.ObterTodos().Count();
+
+            Assert.Equal(2, quantidadeDeBaralhos);
+        }
+
+        [Fact]
+        public void ao_ObterTodos_compara_se_os_primeiros_elementos_sao_iguais()
```

Comment:
não tem necessidade esse teste, se for pra comparar só o primeiro, pode remover ele